packages/odyssey-react-mui/src/Button.tsx

```diff
@@ -164,7 +164,7 @@ const Button = ({
   // We're deprecating the "tertiary" variant, so map it to
   // "secondary" in lieu of making a breaking change
   const variant = variantProp === "tertiary" ? "secondary" : variantProp;
-  const localButtonRef = useRef<HTMLButtonElement>(null);
+  const localButtonRef = useRef<HTMLButtonElement | null>(null);
```

Comment:
We shouldn't need the `| null`; at least, we used to not need it right?

Reply:
I either need to type it as `| null` or, when it's created NOT pass `null` in to be able to reassign it. Otherwise it's read only, according to typescript

Reply:
Checkbox has this:
```
  const localInputRef = useRef<HTMLInputElement>(null);
```

Is it something related to the Tooltip?

Reply:
@KevinGhadyani-Okta Right but, I need to reassign this ref manually like
```js
localInputRef.current = anotherThing
```
Which, you can't do unless you either instantiate is like this 
```js
const localButtonRef = useRef<HTMLButtonElement>();
```
Or type it the way I did here